src/style.css

```diff
@@ -0,0 +1,149 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px 16px;
+}
+
+.card__product {
+  padding-top: 32px;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__image {
+  display: inline-block;
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 40px;
+}
+
+.card__id {
+  margin-bottom: 16px;
+}
+
+.card__info {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 0 4px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__price {
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+  /* margin-top: 20px; */
+  margin-bottom: 16px;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  background-image: url(images/star.svg);
+}
```

Comment:
`4. [TASK] - Make sure to reuse part of [Stars task](https://github.com/mate-academy/layout_stars) here, and keep it operational in case you change block modifier.`

You already did it in `stars` task, by changing modifier `stars--3` to `stars--5` I should got different count of active stars.

always check your task according to task [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)